### PR TITLE
[feature] Adds openwisp-monitoring, openwisp-radius and bug fixes. 

### PR DESCRIPTION
**Pull request by pandafy**
- Adds support for internationalization

### CODE REVIEW

**Comment by pandafy** [comment]
@nemesisdesign do we want to revert this commit https://github.com/openwisp/ansible-openwisp2/pull/320/commits/e91a666abf094abbabfa31bbecd7e1c4f2ddc261 ?

File: README.md [hunk 1]
```diff
@@ -531,6 +531,134 @@ you will update the above playbook as follows:
...
+```
+
+Ensure your `requirements.yml` contains following content:
+
+```yml
+---
+roles:
+  - src: https://github.com/openwisp/ansible-openwisp2.git
+    version: dev
+    name: openwisp.openwisp2-dev
```
Comment (nemesisdesign) [inline README.md:618]: Let's update these instructions so they point to master.

**Review by nemesisdesign — COMMENTED** [review]
> @nemesisdesign do we want to revert this commit [e91a666](https://github.com/openwisp/ansible-openwisp2/commit/e91a666abf094abbabfa31bbecd7e1c4f2ddc261) ?

No, let's keep this, we'll change it once all the modules are ready.

File: README.md (hunk 1 above)
Comment (pandafy) [reply README.md:618]: I woudl keep the name of the role as `openwisp.openwisp2-dev`

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**